mix.exs

```diff
@@ -0,0 +1,29 @@
+defmodule HelloElixir.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :hello_elixir,
+      version: "0.1.0",
+      elixir: "~> 1.12",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
```

Comment:
There is no single call to [`Logger`](https://hexdocs.pm/logger/Logger.html) in the code so I am unsure why would we start its application here.

Reply:
Comented it